### PR TITLE
Update changelog for 1.11 release

### PR DESCRIPTION
**Pull request by mssola**

### CODE REVIEW

File: build/packaging/suseconnect-ng.changes [hunk 1]
```diff
@@ -1,8 +1,14 @@
 -------------------------------------------------------------------
-Tue Aug 15 15:49:20 UTC 2024 - Felix Schnizlein <fschnizlein@suse.com>
-
-- Update version to 1.11 (unreleased)
-  * Add detection for container runtimes.
+Thu Jul 10 12:03:20 UTC 2024 - Miquel Sabaté <msabate@suse.com>
+
+- Update version to 1.11
+  - Added uname as collector (8858fc1d7693, 191d782054a9 and b99d42fda31e)
```
Comment (felixsch) [inline build/packaging/suseconnect-ng.changes:5]: do we need the sha sums for the individual commits?

Comment (mssola) [reply build/packaging/suseconnect-ng.changes:5]: Came for free with the git log command I used 😄 But if you want we can remove it, np

Comment (felixsch) [reply build/packaging/suseconnect-ng.changes:5]: I would remove them, they kinda confusing in the context of a spec file I would argue. But in the end your call!

**Review by felixsch — APPROVED** [review]
lgtm :+1:

File: build/packaging/suseconnect-ng.changes (hunk 1 above)
Comment (mssola) [reply build/packaging/suseconnect-ng.changes]: Removed 😉

**Pull request merged**